common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1423,28 +1419,28 @@
-  "feature.wynntils.worldWaypointDistance.backgroundOpacity.description": "What level of opacity should text background have?",
-  "feature.wynntils.worldWaypointDistance.backgroundOpacity.name": "Background Opacity",
-  "feature.wynntils.worldWaypointDistance.bottomBoundingDistance.description": "How far should off-screen indicator be from bottom of the screen?",
-  "feature.wynntils.worldWaypointDistance.bottomBoundingDistance.name": "Bottom Indicator Distance",
-  "feature.wynntils.worldWaypointDistance.description": "Adds the ability to see the distance to your waypoint in the world.",
-  "feature.wynntils.worldWaypointDistance.horizontalBoundingDistance.description": "How far should off-screen indicator be from two sides of the screen?",
-  "feature.wynntils.worldWaypointDistance.horizontalBoundingDistance.name": "Side Indicator Distance",
-  "feature.wynntils.worldWaypointDistance.maxWaypointTextDistance.description": "At what distance should the waypoint text stop rendering?",
-  "feature.wynntils.worldWaypointDistance.maxWaypointTextDistance.name": "Maximum Distance for Waypoint Text",
-  "feature.wynntils.worldWaypointDistance.name": "Waypoint Distance",
-  "feature.wynntils.worldWaypointDistance.scale.description": "How large should the marker be?",
-  "feature.wynntils.worldWaypointDistance.scale.name": "Scale",
-  "feature.wynntils.worldWaypointDistance.showAdditionalTextInWorld.description": "Whether the name of compass target should be displayed",
-  "feature.wynntils.worldWaypointDistance.showAdditionalTextInWorld.name": "Show name in world",
-  "feature.wynntils.worldWaypointDistance.showAdditonalTextAbove.description": "Whether the additional text should be displayed above the icon instead of bottom",
-  "feature.wynntils.worldWaypointDistance.showAdditonalTextAbove.name": "Show Additional Text Above",
-  "feature.wynntils.worldWaypointDistance.textColor.description": "What color should distance text be?",
-  "feature.wynntils.worldWaypointDistance.textColor.name": "Distance Text Color",
-  "feature.wynntils.worldWaypointDistance.textShadow.description": "What should the text shadow look like?",
-  "feature.wynntils.worldWaypointDistance.textShadow.name": "Text Shadow",
-  "feature.wynntils.worldWaypointDistance.topBoundingDistance.description": "How far should off-screen indicator be from top of the screen?",
-  "feature.wynntils.worldWaypointDistance.topBoundingDistance.name": "Top Indicator Distance",
+  "feature.wynntils.worldMarkers.bottomBoundingDistance.description": "How far should off-screen indicator be from bottom of the screen?",
+  "feature.wynntils.worldMarkers.bottomBoundingDistance.name": "Bottom Indicator Distance",
+  "feature.wynntils.worldMarkers.description": "Adds the ability to see the distance to your waypoint in the world.",
+  "feature.wynntils.worldMarkers.horizontalBoundingDistance.description": "How far should off-screen indicator be from two sides of the screen?",
```

Comment:
Ah, now I understand what this does. I think these need some renaming too. Let me try. We have three variables relating to the top, bottom and sides. They say how far in on the screen the marker should be drawn. 
`screenMarginTop`, `screenMarginBottom` and `screenMarginSides`, perhaps?